src/hooks/useLiveLogs.ts

```
@@ -0,0 +1,408 @@
+    
+    try {
+      // Process the latest log
+      const latestLog = logs[logs.length - 1];
+      const logData = JSON.parse(latestLog);
+      
+      // Only process activity logs
+      if (logData.type === 'activity' && logData.data && logData.data.type) {
+        const activityType = logData.data.type;
+
+        // Get the appropriate handler for this activity type
+        const handler = LOG_HANDLERS[activityType];
```

ESLint

[error] 'LOG_HANDLERS' was used before it was defined.

(@typescript-eslint/no-use-before-define)